development/tools/cmd/rendertemplates/README.md

```diff
@@ -2,18 +2,18 @@
 
 ## Overview
 
-The Render Templates is a tool that reads the configuration in the [`templates/config.yaml`](../../../../templates/config.yaml) file to generate output files, such as Prow component jobs.
+The Render Templates is a tool that reads the configuration from the [`templates/config.yaml`](../../../../templates/config.yaml) file and data files from the [`templates/data`](../../../../templates/data) directory to generate output files, such as Prow component jobs. While `config.yaml` file can hold configuration for an output file, you can place such data within the data files that hold configuration for related output files. Having separate files with grouped data is cleaner and easier to maintain than one huge config file.
 
-The `config.yaml` file specifies the following for the Render Templates:
+The `config.yaml` file and the data files specify the following for the Render Templates:
 - Templates it must use to generate the output files
 - The name and location of the output files
 - Values it must use to generate the output files
 
 ### Data for templates 
 
-The Render Template passes data used to generate files from templates in the **$.Values** and **$.Global** variables. The values of these variables are created from the `config.yaml` file. The **Global** variable holds data from the **global** key. The Render Template generates the values of the **Values** variable from scratch for each Prow job. It is generated by merging ConfigSets. Each ConfigSet is a map defined in one of three possible places in the `config.yaml` file:
+The Render Templates pass data used to generate files from templates in the **$.Values** and **$.Global** variables. The values of these variables are created from the `config.yaml` file and data files stored in the `templates/data` directory. The **Global** variable holds data from the **global** key. The Render Templates generate the values of the **Values** variable from scratch for each Prow job. **Values** variable is generated by merging ConfigSets. Each ConfigSet is a map defined in one of three possible places:
```

Comment:
If you mean it as a tool here, and not as Templates, then it should be singular here. You used both singular and plural verbs throughout the document so I wasn't sure.
I should have clarified this before correction.
```suggestion
The Render Templates passes data used to generate files from templates in the **$.Values** and **$.Global** variables. The values of these variables are created from the `config.yaml` file and data files stored in the `templates/data` directory. The **Global** variable holds data from the **global** key. The Render Templates generates the values of the **Values** variable from scratch for each Prow job. **Values** variable is generated by merging ConfigSets. Each ConfigSet is a map defined in one of three possible places:
```